### PR TITLE
Translated all russian comments to English

### PR DESCRIPTION
**Pull request by viktor-podzigun**
May be some of the comments not relevant any more, but I've just translated them to English.

### CODE REVIEW

**Comment by corporateshark** [comment]
Some translated comments actually do not reflect the original meaning of Russian comments.

File: src/dircalc.cpp [hunk 1]
```diff
@@ -11,7 +11,7 @@
 class OperDirCalcData: public OperData
 {
 public:
-	//после создания эти параметры может трогать толькл поток поиска
+	//after creation these params are touched by search thread only
```
Comment (corporateshark) [inline src/dircalc.cpp]: after creation, these params can be touched by the search thread only

File: src/eloadsave.cpp [hunk 2]
```diff
@@ -187,7 +187,7 @@ void LoadThreadWin::OperThreadStopped()
 
 	file = threadData.file;
 
-	if ( !file.ptr() ) //бывает при  _ignoreENOENT, создаем пустой
+	if ( !file.ptr() ) //may be when _ignoreENOENT, create empty
```
Comment (corporateshark) [inline src/eloadsave.cpp]: can happen if _ignoreENOENT is set, create an empty one in this case

File: src/help.cpp [hunk 3]
```diff
@@ -863,23 +863,23 @@ HelpParzerMem::~HelpParzerMem() {}
...
-   @n - конец строки таблицы
-   @c - конец колонки в строке таблицы
-   @e - конец таблицы
+   @t - table
+   @n - the end of table's row
+   @c - the end of table row's column
+   @e - the end of table
 
-   блок ограничивается символами { и }
+   block surrounded by symbols { and }
```
Comment (corporateshark) [inline src/help.cpp]: block within { and }

File: src/ncwin.h [hunk 4]
```diff
@@ -351,7 +351,7 @@ class NCWin: public NCDialogParent
 	void ExecNoTerminalProcess( const unicode_t* p );
 #endif
 
-	void RightButtonPressed( cpoint point ); //вызывается из панели, усли попало на имя файла/каталого
+	void RightButtonPressed( cpoint point ); //called from panel, if point on file/dir
```
Comment (corporateshark) [inline src/ncwin.h]: called from a panel if the point hits a file/dir

**Comment by corporateshark** [comment]
A lot of information is lost due to this translation. This cannot be merged the way it is done.

**Comment by viktor-podzigun** [comment]
If you can done better, just go ahead. But don't forget, that by behaving like this you may lost motivation in your contributors, since there is no respect to their time and effort. Sorry, but I have to say this.

I'm not an English writer, that's true, so may be my work is not syntactically right in some places, but I think it keeps as close to the original meaning as possible. By providing rephrased variants and saying that they are far better, sorry, but I just cannot agree with you.

Some of the comments are just a piece of shit, and have to be removed. Some of them is even present in exceptions messages, just search for Botva :)

**Comment by corporateshark** [comment]
Sorry, but "time and effort" is not a good reason to merge shitty stuff into the project, I will not do it. Essentially, this PR destroys many useful comments and makes them unreadable.

**Comment by singalen** [comment]
@corporateshark, I believe this is not about rejecting a pull request but rather about the comment wording. It won't hurt to say "thank you" first to people who do an expensive work for free. We don't need to repeat FAR's path in everything :D
Let me thank you all guys for your work, though my part in the work is tiny.
@viktor-podzigun don't forget that there are (and definitely will be!) users who are hugely thankful. This type of software could easily kick `mc` out of its niche, especially if the textmode widgets from wal come to life... I can't wait for it.

**Comment by viktor-podzigun** [comment]
Thank you @singalen I wish everyone behave like you!

-- We don't need to repeat FAR's path in everything :D
That's totally true, BTW, I've doing java plugin system for FAR, but still didn't finished it, may be for the same reason

I don't know whats wrong with @corporateshark today (1 of April :D)

**Comment by helgp** [comment]
*I've doing [..], but still didn't finished[..]*
Your C++ is perfect! :)

**Comment by viktor-podzigun** [comment]
Thanks @helgp on good word.

Will do.

**Comment by corporateshark** [comment]
Guys, it's not about bad or good English, it's about bringing value to the project. I explained why I am not going to merge this PR. Because **it destroys useful information** contained in some original comments. However, you are just saying that I'm doing wrong but never explain what value does this commit bring to the project. And saying that a lot of time and effort was put into something that should not had been done that way in the first place, this is unprofessional.

File: src/color-style.cpp [hunk 5]
```diff
@@ -6,7 +6,6 @@
 
 #include "color-style.h"
 
-//надо нормально разукрасить
```
Comment (corporateshark) [inline src/color-style.cpp]: TODO:

**Comment by singalen** [comment]
@corporateshark we're totally conscious and respect the fact that you're the man makes the whole thing live. Just a notice that a change in a wording, in "sweet wrapping for a bitter pill" style, will keep minor contributors motivated. That will help not only to build a software, but to build a team.

**Comment by corporateshark** [comment]
@singalen However, this does not answer the key question regarding this PR. Building a good product includes making tough decisions like this. If I allow this kind of changes to happen once, it will open the road for other useless things.

**Comment by singalen** [comment]
I believe, everybody is OK with the rejection and there will be a new PR - when it's ready.

**Comment by corporateshark** [comment]
@singalen Just get the idea it is not about the language, it is about useful remarks in the code left by the original author and until this code is refactored/rewritten there is no reason to strip those original comments, they can contain some handy hints.

**Pull request closed without merging**